### DISCUSSION TITLE
customize bootstrap variable & css the right way ?

### DISCUSSION THREAD

**Discussion by asharifauzan** [Laravel-Backpack/community-forum#252] [Q&A (Help)] [answered]
Hello, I'm using Backpack at my work it's great as it is fast to build Admin panel. But the problem I'm facing is to customizing bootstrap color and any other component style. My current approach to do this is by including this repo `https://github.com/DigitallyHappy/BackStrap` at my backpack repo, do `npm run watch`, and including its sass compiled files to base.php but I think this is not the best way since it's gaining asset files and slowing down page load performance. I also working with Ruby on Rails & Bootstrap and it's very easy to customize it. Thanks Anyway.

**Comment by jcastroa87** [comment, marked as answer]
Hello @asharifauzan you can use [Themes](https://backpackforlaravel.com/docs/6.x/base-themes) available on Backpack v6

**Reply by asharifauzan** [reply]
Our project was installed with backpack v5 and we were following bootstrap customization [here](https://getbootstrap.com/docs/4.1/getting-started/theming/) and we weren't sure it was the right way. Anyway thxu for the answer maybe we will use v6 version later.
